images/osl-cli.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 Apache Software Foundation (ASF)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: "packager"
+  version: 1.33.0-snapshot
+  from: "registry.access.redhat.com/ubi8/ubi-minimal:8.9-1161"
+  modules:
+    repositories:
+      - path: modules
+    install:
+      - name: com.redhat.osl.cli.packager
+
+- name: "kn-workflow-cli-artifacts"
+  version: 1.33.0-snapshot
+  description: "Red Hat OpenShift Serverless Logic 1 kn-workflow CLI artifacts"
+  from: "registry.access.redhat.com/ubi8/ubi-minimal:8.9-1161"
+  labels:
+    - name: com.redhat.component
+      value: "openshift-serverless-1-kn-workflow-cli-artifacts-rhel8-container"
+    - name: name
+      value: "openshift-serverless-1/kn-workflow-cli-artifacts-rhel8"
+    - name: version
+      value: "1.33.0-snapshot"
```

Comment:
I dont think we need this "name" and "version" block here, we dont have it for other images. I would drop it to make it simpler.

Reply:
Version is required.

Reply:
so maybe drop the name, I added a suggestion.

Reply:
@rgdoliveira I'm follwing the same approach of the original image. I think we should keep both.

Reply:
please check the other images we productize, I cant see neither `name` nor `version` there. That's why I'm saying that maybe it is better to remove.

Reply:
but anyway, just my opinion, just go with it whatever way you prefer